apps/staging/goproxy/deployment.yaml

```diff
@@ -17,8 +18,8 @@ spec:
         imagePullPolicy: IfNotPresent
         args:
           - "-listen=0.0.0.0:8080"
-          - "-cacheDir=/opt/cache"
-          - "-proxy=https://proxy.golang.org"
+          - "-cacheDir=/data"
+          - "-proxy=https://goproxy.cn" # using qiniu cloud proxy.
```

Comment:
Past experience has shown that goproxy.cn is not as stable as proxy.goproxy.org.